contrib/utilities/run_clang_tidy.sh

```diff
@@ -72,4 +56,15 @@ CC=clang CXX=clang++ cmake $ARGS "$SRC" || (echo "cmake failed!"; false) || exit
 
```

Comment:
In line 55, ShellCheck says
```
                           ^-- SC2086: Double quote to prevent globbing and word splitting.
```
Do you have any idea why 
```
CC=clang CXX=clang++ cmake "$ARGS" "$SRC"
```
doesn't work? I am fine with ignoring.

Reply:
My understanding is that this would give $ARGS as a single argument to cmake (instead of several separated by spaces).